### PR TITLE
[15.0][MIG] base_external_system: Migration to 15.0

### PR DESCRIPTION
**Pull request by cesar-tecnativa**
@Tecnativa TT37714

### CODE REVIEW

**Comment by cesar-tecnativa** [comment]
@pedrobaeza please review

**Comment by cesar-tecnativa** [comment]
@Tardo please review

File: base_external_system/models/external_system_os.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    _inherit = "external.system.adapter"
+    _description = "External System OS"
+
+    previous_dir = None
+
+    def external_get_client(self):
+        """Return a usable client representing the remote system."""
+        super(ExternalSystemOs, self).external_get_client()
+        if self.system_id.remote_path:
+            ExternalSystemOs.previous_dir = os.getcwd()
```
Comment (pedrobaeza) [inline base_external_system/models/external_system_os.py:26]: Why changing this one and not the same on the next method? Anyways, this shouldn't be something stored at server level IMO.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**